src/test/java/bit/javaoop/CompanyTest.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class CompanyTest {
+    @Test
+    void shouldToString() {
+        //given
+        Manager m1 = new FixedBudgedManager("m2", 15001, 20001);
+        AbstractEmployee e1 = new Employee("e1", 100);
+        CEO ceo = new CEO("c1");
+        Company company = new Company("", ceo);
+        String text="";
+        String text2="";
+        //when
+        m1.hire(e1);
+        ceo.addManager(m1);
+        text += ceo.toString() + "\n" +
+                "\t" + m1.toString() + "\n" +
+                "\t\t" + e1.toString() + "\n";
+        //then
+        assertEquals(text,company.toString());
+        assertNotEquals(text2,company.toString());
```

Comment:
Tutaj po pierwsze zrobiłym tak : 
![image](https://user-images.githubusercontent.com/18233796/38472232-34a973fc-3b7d-11e8-9a80-0a3b6837dde9.png)
Następnie rozbiłbym to na dwa testy(metody testowe) bo testujesz tu dwa przypadki w jednej metodzie teraz.
I na koniec jednak ten cały result bym się pofatygował tak żeby go wrzucić do bloku given trochę więcej pisania ale fajnie jak w bloku when jest jedynie akcja, którą testujemy.